### DISCUSSION TITLE
Oliver, can we chat?

### DISCUSSION THREAD

**Discussion by BillBranchRadon** [ozel/DIY_particle_detector#51] [General]
Hey Oliver! I am in the process of creating a video to help the radon measurement community better understand their radon monitor devices. I have opened up every major device for a device tear down. I have many videos and pictures to share. I have a couple of questions I am hoping you can answer when time permits:
1) Disintegrating Radon alpha particle detection: With the professional solid state diodes here is what I have discovered for all of the devices: Counts Per Hour (CPH) is based on chamber size and SSD size 
CPH = chamber size CCs x SSD size (1 sq cm =1) x disintegrations (0.037) x # of chambers
For example: 25CC x 1 sq cm SSD x 0.037 x 4 chambers = 3.7 CPH (this is an actual device)
For example: 45CC x 4 sq cm SSD x 0.037 x 2 chambers = 13.32 CPH (another actual device)
**So, the question becomes** what size of chamber is the max before there are diminishing returns (particle disintegration distance from the SSD)?
2) In the pictures you will see that a common 1 sq cm SSD, an unusual 4 sq cm SSD and a BPX61 that is used in a professional device. It seems that the BPX61 is much smaller, what is the impact of the size in this case?

![2023-03-17_11-19-23](https://user-images.githubusercontent.com/128250860/226126489-7a13d305-4015-4d53-a612-b4a48baa04d5.jpg)
![2022-11-09_08-27-02](https://user-images.githubusercontent.com/128250860/226126502-88321d7c-214a-4b93-a1d9-46b889686fc2.jpg)

**Comment by ozel** [comment]
Dear Bill,

I really do appreciate this sort of exchange and your detailed posts! Keep it coming, even if my response might be delayed...
Myself, I also own an airthings device as well but have yet to dare to open the little pill dose, so thanks for your sneak peek inside. :D

The crucial part about radon measurement is that most devices sample the air based on the physics of air diffusion into a measurement chamber. Meaning the sensor size is only one of many parameters. Chamber volume, geometry and the shape of air vents are equally important. Radon is typically not measured directly, instead, the alpha decay of the following isotope in the decay chain, the decay of polonium, is monitored. Po is often left slightly positively charged after the previous radon decay, which is why it can be collected/guided by a negative voltage. Some radon detectors use this trick to concentrate the polonium isotopes towards the sensor area by means of a small electrical field applied between the sensor and the diffusion chamber. I would say devices which apply such an electric field are more efficient and the more volume enclosed in the diffusion chamber, the better - but this is hard to generalize. I would guess the importance of the sensor size is rather negligible if the diffusion chamber volumes are rather small and in the order of a couple of cubic centimetres. The main advantage should be that larger sensors inside diffusion chambers have an advantage in quicker adaption to a new measurement position. But even in the case of airthings, I remember it takes a couple of weeks/months in the same location, before the accuracy of radon measurements is in a sensible region.

Regarding the BPX61-based Breeze device, I'm curious, how does the chamber look like?

Some more practical radon measurement background is given here: https://www.mdpi.com/1424-8220/19/17/3721

Have you looked at the actual CPH directly of the airthings sensor and compared it with the activity value the algorithm computes based on that? Because this is kind of the secret sauce of that device...
I remember the Bluetooth interface of the airthings wave device does not offer the raw detector counts, one would need to measure somewhere near the sensor electronics. By the way, these devices have also been evaluated and compared in some research studies, if you don't have access to them I may be able to help.

**Reply by BillBranchRadon** [reply]
Awsome! I was starting to give up.
I do not have access to the research studies you mention at the bottom of your reply, I will appreciate the access :-) . I have a trash email I use for public use: **flagstaff2015@ya-oo.co-** (replace hyphens with the obvious characters) 
NOTE: please send to this email and **_I will send you a link_** to device breakdown videos and pictures for the following devices:
All AirThings products (commercial and consumer)
All current SunRadoon devices (1028xp, 1030, and Luft)
All RadStar alpha series devices: a830, a516 and a310
The rad-Elec Recon
Both of the Femto-teck CRMs: LPT and the old workhorse 510)
The RadonEye PRO 

(OK): "Have you looked at the actual CPH directly of the airthings sensor and compared it with the activity value the algorithm computes based on that?"
(BB) No, I would enjoy knowing how to do that. Perhaps I can use that same method with all other devices.

(OK): "Regarding the BPX61-based Breeze device, I'm curious, how does the chamber look like?"
(BB): See attached pictures but the chamber is 1400CCs and the BPX 61 canister edges create large lateral blind spots. 
![BREEZE chamber 1400CC](https://github.com/ozel/DIY_particle_detector/assets/128250860/29d443ec-c335-4507-952e-498016140249)
![BREEZE Tear Down-BPX61 Lense manually removed up close](https://github.com/ozel/DIY_particle_detector/assets/128250860/65ad8339-03e8-48e6-b30c-56831eb6f1fe)
![BREEZE Tear Down-BPX61 Lense manually removed](https://github.com/ozel/DIY_particle_detector/assets/128250860/b84dd2ad-0037-43b9-8d70-2a38de1756fb)
![BREEZE Chamber lid](https://github.com/ozel/DIY_particle_detector/assets/128250860/02ac9e87-df77-4587-aff4-8ac9f53a6242)

**Comment by BillBranchRadon** [comment]
BTW: I am going to try to figure out the maximum size chamber that can be used with a 1cm sq SSD before there are diminishing returns due to larger chamber sizes. I am thinking that the max effective distance from the SSD is 2.5 cm with a 30 degree lateral blind spot on all SSD flat mounting surface edges.
